recipes/libdwarf/all/conandata.yml

```diff
@@ -18,6 +21,10 @@ sources:
     url: "https://github.com/davea42/libdwarf-code/archive/refs/tags/v0.8.0.tar.gz"
     sha256: "8ef0dbfb0816b02aac97b87426689ebaa4efb8ec2ca2c759ea34c5e4678dff3f"
 patches:
+  "2.1.0":
+    - patch_file: "patches/0.9.2-0001-fixes.patch"
+      patch_description: "fix DW_API definition"
```

Comment:
I'm concerned that this patch is still required version after version - is this still needed? Has this been reported upstream?

Reply:
I think that it was solved here: https://github.com/davea42/libdwarf-code/commit/9a856a4bee90c1b7cc3f4e8cf6c2dd8ce6b02848

Reply:
@jcar87 For the record, I'm trying to reopen this issue https://github.com/davea42/libdwarf-code/issues/263
The `off_t` error could hopefully be solved in a future version 🤞

Reply:
I think that [these lines](https://github.com/conan-io/conan-center-index/pull/28032/files#diff-78304d59106a97f3259b5f80b4d245b025f314d884ea53fd37c8d8de5146cd16R111-R113) save us from adding that patch.